### ISSUE TITLE
"Unknown word" error in SCSS syntax and interpolated value with function

### ISSUE THREAD

**Issue by niksy** [postcss/postcss-scss#36]
This issue was first submitted to https://github.com/stylelint/stylelint/issues/1640 but I’m c/ping whole description for easier discussion.

---

## Bug description

When trying to use function inside SCSS interpolation, stylelint reports syntax error with message "Unknown word".

```sh
index.scss
 17:44  ✖  Unknown word   CssSyntaxError
```

## How to reproduce

Input SCSS:

```scss
.foo {
	$var:foo;
	background-image:#{$var}bar;
}

.bar {
	$var:to-upper-case(foo);
	background-image:#{$var}bar;
}

.bad {
	$var:to-upper-case(foo);
	background-image:url(#{$var}bar);
}

.baz {
	background-image:url(#{to-upper-case(foo)}bar); // This is where error is triggered
}
```

Output CSS with node-sass 3.8.0:

```css
.foo {
  background-image: foobar; }

.bar {
  background-image: FOObar; }

.bad {
  background-image: url(FOObar); }

.baz {
  background-image: url(FOObar); }
```

---

Using latest stylelint with SCSS parser.

**Comment by ai** [comment]
Seems like it is postcss-scss. I will try to look today.

**Comment by ai** [comment]
OK. Problem is because of `url()` — it has special logic. I am trying to find way how to solve it in best way :).

**Comment by niksy** [comment]
I suppose this goes to other CSS functions like `calc()` and such? Haven't tried it.

**Comment by ai** [comment]
@niksy nope, `url()` is not regular function in CSS, it has very special semantic. Even in Sass is ti not a regular function. `url(http://example.com)` — because of `//`, `example.com` should be a comment in Sass, but it doesn’t — because `url()` has different logic.

I will find solution, but I need time.

**Comment by ai** [comment]
Fixed https://github.com/postcss/postcss-scss/commit/01bb9da154cd3ee97932774d2ea3c26099b50a72

**Comment by ai** [comment]
I will release it, when I will fix this one too https://github.com/postcss/postcss-scss/issues/38

**Comment by ai** [comment]
Released in 0.1.9
